.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Set up Node
         uses: actions/setup-node@v3
         with:
-          node-version: "17"
+          node-version: "18"
```

Comment:
Would it be viable to leave this out so it always uses the latest major Node release?

Reply:
(but I suppose we'd want it to match `dev.yml`).

Reply:
I think we're better off using stable releases, which are the even numbers. For example, version 19 is already out, but it's an unstable release.